operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md

```diff
@@ -74,14 +74,13 @@ make sure that the following conditions are met:
 
 ### Master node
 
-Master node rebuilds require that the environment variables `CSM_RELEASE`, `CSM_REL_NAME`, and `CSM_ARTI_DIR` be set on the node where the rebuild script is executed.
+Master node rebuilds require that the environment variables `CSM_RELEASE` and `CSM_ARTI_DIR` be set on the node where the rebuild script is executed.
 
-(`ncn-m#`) Set the `CSM_RELEASE`, `CSM_REL_NAME`, and `CSM_ARTI_DIR` environment variables. Replace `1.4.0` with the correct CSM release version:
+(`ncn-m#`) Set the `CSM_RELEASE` and `CSM_ARTI_DIR` environment variables. Replace `1.4.0` with the correct CSM release version:
 
 ```bash
 export CSM_RELEASE=1.4.0
-export CSM_REL_NAME=csm-${CSM_RELEASE}
-export CSM_ARTI_DIR="/etc/cray/upgrade/csm/${CSM_REL_NAME}/tarball/${CSM_REL_NAME}"
+export CSM_ARTI_DIR="/etc/cray/upgrade/csm/csm-${CSM_RELEASE}/tarball/${CSM_RELEASE}"
```

Comment:
I think you'll need to adjust this too, if you're going to make this change: https://github.com/Cray-HPE/docs-csm/blob/main/upgrade/scripts/upgrade/prepare-assets.sh#L80-L81

Reply:
Missing the `^csm-` change on the last var replacement.

Reply:
I'm missing the `^csm-` change on the last var replacement.

The link you posted is fine, the script defines CSM_REL_NAME so it's transparent to the admin. I'm trying to keep changes to a minimum here, just where UX is affected.